dashboard/app/controllers/api/v1/pd/course_facilitators_controller.rb

```diff
@@ -3,8 +3,7 @@ class Api::V1::Pd::CourseFacilitatorsController < ApplicationController
 
   # GET /api/v1/pd/course_facilitators
   def index
-    course = params.require(:course)
-    facilitators = Pd::CourseFacilitator.facilitators_for_course course
+    facilitators = params.has_key?(:course) ? Pd::CourseFacilitator.facilitators_for_course(params.require(:course)) : Pd::CourseFacilitator.all.map(&:facilitator)
```

Comment:
I find these long lines (this is 161 cols) hard to read, and it goes against our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#all-languages) limit
> Stick to an 80 character line length when possible, except for long URLs in comments or long regular expressions. Occasionally wiggling between 80 and 120 characters is OK in cases where it greatly improves the readability over a multiple line split.

Please split up for readability,.